### ISSUE TITLE
Longformer global attention mask, 2 or 1? 

### ISSUE THREAD

**Issue by kakeith** [huggingface/transformers#7015]
This example in the LongFormer documentation (https://huggingface.co/transformers/model_doc/longformer.html )
```
attention_mask[:, [1, 4, 21,]] = 2  # Set global attention based on the task. For example,
...                                     # classification: the <s> token
...                                     # QA: question tokens
...                                     # LM: potentially on the beginning of sentences and paragraphs
```
seems to mislead the user to put a 2 as the global attention mask instead of a 1 this is described in the actual documentation. I'm guessing this is a mistake because (a) the example is directly copied from the AllenAI (https://github.com/allenai/longformer#how-to-use), and (b) the source code actually adds a +1 to the attention mask in the function `_merge_to_attention_mask` (https://huggingface.co/transformers/_modules/transformers/modeling_longformer.html#LongformerModel.forward). 

Could the documentation please be fixed or clarified? Thanks! 

Also what is the 1, 4, 21 in that example? 

### Who can help
 Longformer/Reformer: @patrickvonplaten

## Information

Model I am using (Bert, XLNet ...): LongFormer

**Comment by patrickvonplaten** [comment]
That is 100% correct - thanks for notifying us :-) PR is linked.

**Comment by pratikchhapolika** [comment]
Also what is the 1, 4, 21 in that example? Can someone answer this?

**Comment by patrickvonplaten** [comment]
Just random indices that will be attended and will attend globally

**Comment by pratikchhapolika** [comment]
So if we are passing a sentence during inferencing `I love Hugging face` and I give value attention_mask[:, [0,-1]] = 2  then ` <s> and </s>` will be attended globally?? And then I can fetch their embedding to get embedding of the sentence?
